models/pose_estimation_mediapipe/demo.cpp

```diff
@@ -0,0 +1,2860 @@
+                hconcat(landmarksWord.col(0), landmarksWord.col(2), landmarksXZ);
+                landmarksXZ.col(1) = -landmarksXZ.col(1);
+                x = landmarksXZ * 100;
+                x.col(0) += 300;
+                x.col(1) += 100;
+                x.convertTo(landmarksXZ, CV_32S);
+                drawLines(display3d, landmarksXZ, keepLandmarks, true, 2);
+
+                // Left view
+                Mat landmarksYZ;
+                hconcat(landmarksWord.col(2), landmarksWord.col(1), landmarksYZ);
+                landmarksYZ.col(0) = -landmarksYZ.col(0);
+                x = landmarksYZ * 100;
+                x.col(0) += 100;
+                x.col(1) += 300;
+                x.convertTo(landmarksYZ, CV_32S);
+                drawLines(display3d, landmarksYZ, keepLandmarks, true, 2);
+
+                // Right view
+                Mat landmarksZY;
+                hconcat(landmarksWord.col(2), landmarksWord.col(1), landmarksZY);
+                x = landmarksZY * 100;
+                x.col(0) += 300;
+                x.col(1) += 300;
+                x.convertTo(landmarksZY, CV_32S);
+                drawLines(display3d, landmarksZY, keepLandmarks, true, 2);
+            }
+        }
+    }
+    return pair<Mat, Mat>(displayScreen, display3d);
+}
+
+
+
+int main(int argc, char** argv)
+{
+    CommandLineParser parser(argc, argv, keys);
+ 
+    parser.about("Person Detector from MediaPipe");
+    if (parser.has("help"))
+    {
+        parser.printMessage();
+        return 0;
+    }
+
+    string model = parser.get<String>("model");
+    float confThreshold = parser.get<float>("conf_threshold");
+    float scoreThreshold = 0.5f;
+    float nmsThreshold = 0.3f;
+    int topK = 5000;
+    bool vis = parser.get<bool>("vis");
+    bool save = parser.get<bool>("save");
+    int backendTargetid = parser.get<int>("backend");
+
+    if (model.empty())
+    {
+        CV_Error(Error::StsError, "Model file " + model + " not found");
+    }
+    VideoCapture cap;
+    if (parser.has("input"))
+        cap.open(samples::findFile(parser.get<String>("input")));
+    else
+        cap.open(0);
+    Mat frame;
+    // person detector
+    MPPersonDet modelNet("../person_detection_mediapipe/person_detection_mediapipe_2023mar.onnx", nmsThreshold, scoreThreshold, topK,
+        backendTargetPairs[backendTargetid].first, backendTargetPairs[backendTargetid].second);
+    // pose estimator
+    MPPose poseEstimator(model, confThreshold, backendTargetPairs[backendTargetid].first, backendTargetPairs[backendTargetid].second);
+    //! [Open a video file or an image file or a camera stream]
+    if (!cap.isOpened())
+        CV_Error(Error::StsError, "Cannot opend video or file");
+
+    static const std::string kWinName = "MPPose Demo";
+    while (waitKey(1) < 0)
+    {
+        cap >> frame;
+        if (frame.empty())
+        {
+            cout << "Frame is empty" << endl;
+            waitKey();
+            break;
```

Comment:
I think we can use `continue;` and remove these 2 lines. Because sometimes the camera will get empty frame when it just turn on. Using `continue;` will not block the process.